src/components/Confirm/PdfPreview.js

```diff
@@ -0,0 +1,67 @@
+import { h, Component } from 'preact'
+import PDFObject from 'pdfobject'
+import {preventDefaultOnClick} from '../utils'
+import style from './style.css'
+
+const IEPdfBlobLink = ({blob}) => {
```

Comment:
you might want to explain in a comment why this component is used, as far as I can remember is because IE does not work with generated object URLs on links.